erpnext/patches/v12_0/update_vehicle_maintenance_schedule.py

```diff
@@ -0,0 +1,20 @@
+import frappe
+
+
+def execute():
```

Comment:
We'll have to remove the patch and use it as a script instead .. since we cant set data due after data in Project Templates until we update ...